seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/java/org/apache/seatunnel/e2e/flink/v2/iotdb/FakeSourceToIoTDBIT.java

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.seatunnel.e2e.flink.v2.iotdb;
 
-import static org.awaitility.Awaitility.given;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.given;
```

Comment:
revert?

Reply:
It has some compile problems in my local environment, so I changed it.

Reply:
try clean local environment？